akka-docs/rst/java/code/docs/stream/ActorSubscriberDocTest.java

```diff
@@ -170,18 +172,26 @@ public WorkerPool() {
 
             if (queue.size() > MAX_QUEUE_SIZE)
               throw new RuntimeException("queued too many: " + queue.size());
-                  
+
             router.route(WorkerPoolProtocol.work(msg.id), self());
         }).
+        match(ActorSubscriberMessage.onCompleteInstance().getClass(), complete -> {
```

Comment:
Wouldn't `matchEquals(ActorSubscriberMessage.onCompleteInstance(), ...` be nicer (and more aligned with what you'd do in Scala) here?